### PR TITLE
Remove GitBook instructions

### PR DESCRIPTION
**Pull request by parkerbxyz**
These instructions are no longer applicable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,13 +47,6 @@ script/server
 
 Docsify doesn't currently support creating PDFs, but you can generate one using [GitBook legacy](https://docs.gitbook.com/resources/gitbook-legacy). We've tried to make this simple for you by creating [`script/generate-pdf`](script/generate-pdf).  The legacy version of GitBook is being shut down soon, but it still works.  You'll need to follow the installation instructions below to install GitBook.  The previous installation instructions on the GitBook site have been removed.
 
-## GitBook dependency
-
-1. Download and install [Calibre](https://calibre-ebook.com/download).
```
Comment (parkerbxyz) [inline README.md:52]: Example comment

File: README.md [hunk 2]
```diff
@@ -47,13 +47,6 @@ script/server
...
-
-1. Run Calibre once to set up the application.
-
-1. Make sure that the [Calibre command line interface](https://manual.calibre-ebook.com/generated/en/cli-index.html) is available on your path.  This is located at `/Applications/calibre.app/Contents/MacOS/` on macOS if you are running the application from the default location.
```
Comment (parkerbxyz) [inline README.md:56]: Another comment

File: README.md [hunk 3]
```diff
@@ -47,13 +47,6 @@ script/server
...
```
Comment (parkerbxyz) [inline README.md:50]:
```suggestion
```

**Review by parkerbxyz — COMMENTED** [review]
LGTM 👍

**Review by michaelsainz — APPROVED** [review]
👍

**Pull request merged**